File: API/Entities/Product.cs
using System.Collections.Generic;

namespace API.Entities
{
    public class Product
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string ProductPrice { get; set; }
        public string SalePrice { get; set; }
        public string Description { get; set; }
        public string Category { get; set; }
        public int Stock { get; set; }
        public bool Highlight { get; set; }
        public ICollection<ProductPhoto> Photos { get; set; }
        public virtual ICollection<OrderedProducts> OrderedProducts { get; set; }
    }
}

File: client/src/app/_models/product.ts
import { ProductPhoto } from "./productPhoto";

export interface Product {
    id: number;
    name: string;
    photoUrl: string;
    productPrice: string;
    salePrice: string;
    description: string;
    category: string;
    stock: number;
    highlight: boolean;
    photos: ProductPhoto[];
}